nilearn/plotting/displays.py

```diff
@@ -764,7 +768,7 @@ def _show_colorbar(self, cmap, norm, threshold=None):
         istart = int(norm(-offset, clip=True) * (our_cmap.N - 1))
```

Comment:
More descriptive names for istart & istop would be helpful. Like why these are the start & stop values. 
A comment explaining what is being done here would be helpful.